src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md

```diff
@@ -246,6 +246,16 @@ To set attributes for the HTML `body` tag use the `<attribute>` instruction.
 </page>
 ```
 
+Also from admin site __Layout Update XML__ field of __Design__ section on _Content Edit_ page  set attributes for the HTML `body` tag use the `<attribute>` instruction.
```

Comment:
I'm not sure what you are trying to do by adding this information here. Can you provide more information about he reason you want to include this update?

Reply:
Hello @meker12,
This information is for adding custom class in `<body>` from admin, No need to add code in `XML `or `PHTML `